src/libraries/System.Reflection.Metadata/ref/System.Reflection.Metadata.cs

```diff
@@ -1731,13 +1725,13 @@ public sealed partial class MetadataReaderProvider : System.IDisposable
     {
         internal MetadataReaderProvider() { }
         public void Dispose() { }
-        public static unsafe System.Reflection.Metadata.MetadataReaderProvider FromMetadataImage(byte* start, int size) { throw null; }
+        public unsafe static System.Reflection.Metadata.MetadataReaderProvider FromMetadataImage(byte* start, int size) { throw null; }
```

Comment:
Not seems like intended change

Reply:
Again, this is an auto-generated change. I'm hesitant to change it back since it will get changed again next time we regenerate the signatures.

Reply:
Should we fix the generator? @stephentoub I found a lot of "static unsafe" https://github.com/dotnet/runtime/search?l=C%23&q=%22static+unsafe%22 and very few "unsafe static" https://github.com/dotnet/runtime/search?l=C%23&q=%22unsafe+static%22.

It seems like we want "static unsafe"?

Reply:
Yeah, let's generate whatever pattern is currently predominant.